### ISSUE TITLE
The Need for Simple Presence on RosettaCode

### ISSUE THREAD

**Issue by appcypher** [simple-lang/simple#20]
[RosettaCode](http://www.rosettacode.org) is a repository where codes implementing tasks in different languages are stored. Simple has no presence there yet. So I think we should fix that.

I already created a [SimpleLang category](http://www.rosettacode.org/wiki/Category:SimpleLang) since `Simple` isn't available as category. Nothing is there yet.
Anyone can signup to the site and contribute to it. 

An example of a tool that use codes on rosetta is `http://rosetta.alhur.es/`, which allows you to compare codes in different languages. An example of this is `http://rosetta.alhur.es/compare/python/java`

This will allow programmers from other languages to get a feel of what idiomatic Simple code looks like compared to a language they are used to.

----------------------------------------------------------
More information on how to add a language to RosettaCode http://www.rosettacode.org/wiki/Rosetta_Code:Add_a_Language

**Comment by Youngestdev** [comment]
Yes, it won't be closed anytime soon. We need to discuss this. I'll dm you
straight away.

**Comment by Youngestdev** [comment]
I think snippets would be released once we hit the version 0.4. well have a well structured syntax and algorithmic methods

**Comment by Thecarisma** [comment]
The simple version 0.4.0 has been released now Documentation on public domains such as wikis, snippets site can be made to increase awareness of the project
